server_manager/model/server.ts

```diff
@@ -186,10 +186,31 @@ export interface AccessKey {
   dataLimit?: DataLimit;
 }
 
-export type BytesByAccessKey = Map<AccessKeyId, number>;
-
 // Data transfer allowance, measured in bytes.
 // NOTE: Must be kept in sync with the definition in src/shadowbox/access_key.ts.
 export interface DataLimit {
   readonly bytes: number;
 }
+
+export type ServerMetricsJson = {
+  server: {
+    location: string;
+    asn: number;
+    asOrg: string;
+    tunnelTime?: {
+      seconds: number;
+    };
+    dataTransferred?: {
```

Comment:
Rename DataLimit to Data, and reuse in the fields here. Wait, we have DataAmount.
Ugh, this needs to be cleaned up. Please add a TODO